### ISSUE TITLE
Accessibility : set ViewCompat.setAccessibilityDelegate() in a safe manner 

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10964]
Related to work being carried out in #10921
### Expected behavior
That when using `ViewCompat.setAccessibilityDelegate` there should be a check that ensures another delegate isn't currently set because future accessibility improvements could overwrite them and break existing behavior. 

### Actual behavior
`ViewCompat.setAccessibilityDelegate` is used on a `View` without creating any checks. 

### Steps to reproduce the behavior
1. Simply do a search of the repository for `ViewCompat.setAccessibilityDelegate`
2. You will see that no checks have been made on the current state of the `View`s  delegate.
